.github/workflows/sync_crowdin.yml

```diff
@@ -38,16 +40,15 @@ jobs:
       - name: Remove empty values from JSON
         run: |
           cd assets/i18n
-          sudo chmod 777 *
           dart nuke.dart >> $GITHUB_STEP_SUMMARY
 
-      - name: Push out changes to i10n
+      - name: Commit translations
         run: |
           git config user.name revanced-bot
           git config user.email github@revanced.app
           sudo chown -R $USER:$USER .git
           git add assets/i18n/*.json
-          git commit -m "chore(Translation): Remove empty values from JSON" assets/i18n/*.json
+          git commit -m "chore: Remove empty values from JSON" assets/i18n/*.json
```

Comment:
I am seeing this commit after every push to dev. Not sure what this is for but maybe a proper trigger can be made to reduce this spam @Ushie

Reply:
It is pushed to the translation branch before it is squash and merged to dev, this is to remove empty string from JSON to make them cleaner.

I guess we could squash these two before pushing.

Reply:
Why would empty strings be present?

Reply:
> Why would empty strings be present?

Not sure, when we exported them from Crowdin

It would look like this

![Screenshot_20240226_104456_GitHub.png](https://github.com/ReVanced/revanced-manager/assets/93124920/7d79eb00-35cc-4a81-8671-1baaa4c241ee)

We decided that it isn't really supposed to be doing that (and it could mess everything up with the language feature), so we decided to work on a postprocessor for it.

I have haven't really decided on a name so I called them Nuke:tm:, a bare-bone (probably not even finished) documentation exist here: https://github.com/ReVanced/revanced-manager/blob/dev/assets%2Fi18n%2FREADME.md

Reply:
> Why would empty strings be present?

Because Crowdin doesn't have a post-processor for nested JSON, so everything that isn't translated is exported as an empty string, no matter the settings in Crowdin

Reply:
> I am seeing this commit after every push to dev. Not sure what this is for but maybe a proper trigger can be made to reduce this spam @Ushie

We can change the trigger of translation sync to manual or to a CRON job

Reply:
> Because Crowdin doesn't have a post-processor for nested JSON, so everything that isn't translated is exported as an empty string, no matter the settings in Crowdin

When does this occur? When stings are deleted?